sdk/identity/azure-identity/src/test/resources/settings.json

```diff
@@ -0,0 +1,15 @@
+{
+    "editor.suggestSelection": "first",
+    "vsintellicode.modify.editor.suggestSelection": "automaticallyOverrodeDefaultValue",
+    "window.zoomLevel": 0,
+    "java.errors.incompleteClasspath.severity": "ignore",
+    "workbench.startupEditor": "newUntitledFile",
+    "python.jediEnabled": false,
+    "java.configuration.checkProjectSettingsExclusions": false,
+    "java.requirements.JDK11Warning": false,
+    "java.semanticHighlighting.enabled": true,
+    "java.home": "/Contents/Home",
+    "python.languageServer": "Microsoft",
+  // this is a dummy comment.
```

Comment:
JSON does not support comments - this may cause some unexpected behaviors?

Reply:
The azure maven plugin requested the feature to allow reading JSON files with comments in them for VS Code settings file.
So, the object mapper is configured to allow reading JSON files with comments in them.